### PR TITLE
Add documentation about the combined use of `$globals` and `$runtime_params`

### PR DESCRIPTION
**Pull request by ElenaKhaustova**
## Description
Solves #3706 

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [x] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Updated the documentation to reflect the code changes
- [x] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [x] Added tests to cover my changes
- [x] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: docs/source/configuration/advanced_configuration.md [hunk 1]
```diff
@@ -218,6 +218,31 @@ companies:
...
+```yaml
+my_global_value: 4
+```
+
+This will allow you to pass a runtime parameter named `random` through the CLI to specify the value of `model_options.random_state` in your project's parameters:
+```bash
+kedro run --params random=3
+```
+
+If the `random` parameter is not passed through the CLI `--params` option with `kedro run`, then `my_global_value` from `globals.yaml` is used for the `model_options.random_state`.
```
Comment (noklam) [inline docs/source/configuration/advanced_configuration.md:244]:
```suggestion
If the `random` parameter is not passed through the CLI `--params` option with `kedro run`, then `my_global_value` from `globals.yml` is used for the `model_options.random_state`.
```

File: docs/source/configuration/advanced_configuration.md [hunk 2]
```diff
@@ -218,6 +218,31 @@ companies:
...
+
+However, you can use `globals` and `runtime_params` by specifying `globals` as a default value to be used in case the runtime parameter is not passed.
+
+Consider this `parameters.yml`:
+```yaml
+model_options:
+  random_state: "${runtime_params:random, ${globals:my_global_value}}"
+```
+
+and this `globals.yaml` file:
```
Comment (noklam) [inline docs/source/configuration/advanced_configuration.md:233]:
```suggestion
and this `globals.yml` file:
```

**Review by noklam — APPROVED** [review]
There are some changes in pyproject.toml which I assume not related to this PR?

**Review by noklam — APPROVED** [review]

**Review by astrojuanlu — APPROVED** [review]
💯 Thanks @ElenaKhaustova !

**Pull request merged**